src/uwtools/drivers/forecast.py

```diff
@@ -184,31 +116,155 @@ def run(self, cycle: datetime) -> bool:
+        """
+        Calls the methods for the set of configuration files neeed by the given model.
+
+        :param cycle: The date and start time for the forecast.
+        :param run_directory: Path of desired run directory.
+        """
+
+
+class FV3Forecast(Forecast):
+    """
+    A driver for the FV3 forecast model.
+    """
+
+    def __init__(
+        self,
+        config_file: str,
+        dry_run: bool = False,
+        batch_script: Optional[str] = None,
+    ):
+        """
+        Initialize the Forecast Driver.
+        """
+
+        super().__init__(config_file=config_file, dry_run=dry_run, batch_script=batch_script)
+
+    # Public methods
+
+    @classmethod
+    def create_directory_structure(
+        cls, run_directory: DefinitePath, exist_act: str = "delete"
+    ) -> None:
+        """
+        Creates the run directory and adds subdirectories INPUT and RESTART. Verifies creation of
+        all directories.
+
+        :param run_directory: Path of desired run directory.
+        :param exist_act: Could be any of 'delete', 'rename', 'quit'. Sets how the program responds
+            to a preexisting run directory. The default is to delete the old run directory.
+        """
+        run_directory = Path(run_directory)
+        cls._create_run_directory(run_directory, exist_act)
+        # Create the two required subdirectories.
+        for subdir in ("INPUT", "RESTART"):
+            path = run_directory / subdir
+            log.info("Creating directory: %s", path)
+            os.makedirs(path)
+
+    def create_field_table(self, output_path: OptionalPath) -> None:
+        """
+        Uses the forecast config object to create a Field Table.
+
+        :param output_path: Optional location of output field table.
+        """
+        self._create_user_updated_config(
+            config_class=FieldTableConfig,
+            config_values=self._config.get("field_table", {}),
+            output_path=output_path,
+        )
+
+    def create_model_configure(self, cycle: datetime, output_path: OptionalPath) -> None:
+        """
+        Uses the forecast config object to create a model_configure.
+
+        :param cycle: The date and start time for the forecast.
+        :param output_path: Optional location of the output model_configure file.
+        """
+        self._create_user_updated_config(
+            config_class=YAMLConfig,
+            config_values=self._config.get("model_configure", {}),
+            output_path=output_path,
+        )
+        start_time = cycle.strftime("%Y-%m-%d_%H:%M:%S")
+        date_values = {
+            "start_year": cycle.strftime("%Y"),
+            "start_month": cycle.strftime("%m"),
+            "start_day": cycle.strftime("%d"),
+            "start_hour": cycle.strftime("%H"),
+            "start_minute": cycle.strftime("%M"),
+            "start_second": cycle.strftime("%S"),
+        }
+        log.info(f"Updating namelist date values to start at: {start_time}")
```

Comment:
Is `model_configure` the file that is kind-of-but-not-really a namelist file? If so, should we say something other than "namelist" here? Or, even if I'm wrong about that, might it be helpful to log that this is about the `model_configure` file, or even e.g.
```
log.info(f"Updating date values to start at {start_time} in {output_path}")
```
?